modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -110,6 +110,12 @@
 #include "NewmarkAccelAux.h"
 #include "NewmarkVelAux.h"
 
+#include "CavityPressureAction.h"
+#include "CavityPressurePostprocessor.h"
+#include "CavityPressurePPAction.h"
+#include "CavityPressureUserObject.h"
+#include "CavityPressureUOAction.h"
```

Comment:
We group the includes by the respective system type (just like the register calls below)